include/rocksdb/version.h

```diff
@@ -5,7 +5,7 @@
 #pragma once
 
 #define ROCKSDB_MAJOR 6
-#define ROCKSDB_MINOR 2
+#define ROCKSDB_MINOR 3
```

Comment:
6.2.2, right?

Reply:
Oh, never mind, I see you're changing master to 6.3.0.

Reply:
I am under the impression that after a major version release, we should bump the version in master to the next major version number.

Reply:
Chatted offline, keep this as 6.2.0 for now.